src/paradigma/ppg_preprocessing.py

```diff
@@ -288,23 +289,23 @@ def synchronization(ppg_meta, imu_meta):
 
     return segment_ppg_total, segment_imu_total
 
-def extract_overlapping_segments(df_ppg, df_imu, time_column_ppg='time', time_column_imu='time'):
+def extract_overlapping_segments(df_ppg, df_acc, time_column_ppg='time', time_column_imu='time'):
```

Comment:
Don't forget to set data types here too.

Reply:
This is not up to date anymore I guess since we are combining stuff in the other PR?